content/en/architectures/adobe/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Scaling Adobe’s Service Delivery Foundation with a Cell-based Architecture
+date: 2024-10-11
+org_name: Adobe
+org_team: Developer Platforms
+org_url: https://www.adobe.com/
+contact: Vikram Sethi
+email: vik.set@gmail.com
+org_description: |
+
+org_size:  10,000+
+user_size:  14,000+
+industries:
+- 
```

Comment:
Similar thing here, if you could add a few industries that would be round things out to be fully complete :tada: